app/views/layouts/application.html.erb

```diff
@@ -10,14 +10,27 @@
 
   <body>
     <nav class = "topnav">
+      <%= link_to "Home Page", "/" %>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Register", '/register' %>
-      <% if current_user %>
+      <% if current_user && current_user.merchant? %>
```

Comment:
Let's clarify if there is a better way to call this (per our method in the application controller).

Reply:
Yes, let's ask during project check-in. Great reminder!